bc_obps/registration/tests/utils/baker_recipes.py

```diff
@@ -101,3 +102,22 @@
     meets_notification_to_director_on_criteria_change=False,
 )
 contact = Recipe(Contact, business_role=BusinessRole.objects.first())
+
+
+# transfer event bakers
```

Comment:
Decision was not show to show transfer event contacts at this time, but I'll leave the recipes in case someone else is testing transfer events more generally